libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -313,6 +323,24 @@ pub fn interpret_ballot_card(
         None => ImageDebugWriter::disabled(),
     };
 
+    [
+        (SIDE_A_LABEL, &side_a, &mut side_a_debug),
+        (SIDE_B_LABEL, &side_b, &mut side_b_debug),
+    ]
+    .par_iter_mut()
+    .map(|(label, side, debug)| {
+        let streaks = detect_vertical_streaks(&side.image, side.threshold, debug);
```

Comment:
Do we think this should be optional? I'm wondering whether there should be an escape hatch to turn this feature off in case of something we don't foresee causing false positives (maybe scanner image sensor failure?). I can also see the argument that if there really are issues with the image that match streaks that we really don't want to interpret them. A question for @mattroe and @benadida, probably.

Reply:
great question. Can we test and see how likely it is to get a false positive before we do extra work to make it togglable? I can be convinced either way easily.

Reply:
It's a good question! I'm gonna go ahead and merge this PR and we can discuss in Slack. I'd prefer to implement an escape hatch for this as a follow up anyway